core/src/main/scala/org/bitcoins/core/api/dlcoracle/DLCOracleApi.scala

```diff
@@ -85,4 +85,9 @@ trait DLCOracleApi {
 
   /** Signs the SHA256 hash of the given bytes using the oracle's signing key */
   def signMessage(message: ByteVector): SchnorrDigitalSignature
+
+  /** Backup oracle database
+    * @param location baclup file location
+    */
+  def backup(location: String): Future[Unit]
```

Comment:
Any reason to to change location to a `Path` rather than `String`? Also it is probably worth noting in the scaladoc this is effictively a noop for postgres backends and only works for sqlite